packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -333,6 +334,19 @@ const resetPrompt = (event) => {
   }
 }
 
+const installCommand = computed(() => {
+  switch (props.gql.currentProject?.packageManager) {
+    case 'npm':
+      return 'npm install --save-dev '
+    case 'yarn':
+      return 'yarn add --dev '
```

Comment:
Super small, but we use `yarn add -D` or `npm install -D` in the ManualInstall step.

Reply:
you got it ;)

Reply:
8b18a9d22ce70e8754d2cf0c8d779a5e8d884e08